pom.xml

```diff
@@ -76,101 +82,63 @@
-        </executions>
-      </plugin>
-      <plugin>
-        <!-- see http://davidb.github.com/scala-maven-plugin -->
-        <groupId>net.alchim31.maven</groupId>
-        <artifactId>scala-maven-plugin</artifactId>
-        <version>3.3.2</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>compile</goal>
-              <goal>testCompile</goal>
-            </goals>
-            <configuration>
-              <args>
-                <arg>-dependencyfile</arg>
-                <arg>${project.build.directory}/.scala_dependencies</arg>
-              </args>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
       <plugin>
         <groupId>io.fabric8</groupId>
         <artifactId>docker-maven-plugin</artifactId>
         <version>0.27.2</version>
         <configuration>
           <images>
             <image>
-              <!-- what is this version? -->
-              <name>bootstrap-demo-kubernetes-api:1.3.3.7</name>
+              <name>%a</name>
               <build>
+                <tags>
+                  <tag>${version.number}</tag>
+                  <tag>latest</tag>
+                </tags>
                 <from>openjdk:8-jre-alpine</from>
                 <ports>
                   <port>8080</port>
                   <port>8558</port>
-                  <port>2552</port>
                 </ports>
                 <entryPoint>
-                  <exec>
-                    <args>/bin/sh</args>
-                    <args>-c</args>
-                    <args>java -jar /maven/cluster-bootstrap-demo-kubernetes-api-java-allinone.jar</args>
-                  </exec>
+                  java $JAVA_OPTS -cp '/maven/*' akka.cluster.bootstrap.demo.DemoApp
                 </entryPoint>
                 <assembly>
-                  <inline>
-                    <dependencySets>
-                      <dependencySet>
-                        <useProjectAttachments>true</useProjectAttachments>
-                        <includes>
-                          <include>com.lightbend:cluster-bootstrap-demo-kubernetes-api-java:jar:allinone</include>
-                        </includes>
-                        <outputFileNameMapping>cluster-bootstrap-demo-kubernetes-api-java-allinone.jar</outputFileNameMapping>
-                      </dependencySet>
-                    </dependencySets>
-                  </inline>
+                  <descriptorRef>artifact-with-dependencies</descriptorRef>
                 </assembly>
               </build>
             </image>
           </images>
         </configuration>
+        <executions>
+          <execution>
+            <id>build-docker-image</id>
+            <phase>package</phase>
+            <goals>
+                <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+          <groupId>pl.project13.maven</groupId>
```

Comment:
git based versioning

Reply:
Oh hey there 👋 Nice to see the plugin here 😄

Reply:
👍 thanks for the plugin @ktoso